src/index.ts

```diff
@@ -137,4 +145,44 @@ export default class BugzillaAPI {
 
     return this.searchBugs(searchParams);
   }
+
+  public async getComment(commentId: number) {
+    let comment = await this.link.get(
+      `bug/comment/${commentId}`,
+      object(CommentsSpec),
+    );
+
+    if (!comment) {
+      throw new Error(`Failed to get comment #${commentId}.`);
+    }
+
+    return Object.fromEntries(comment.comments);
+  }
+
+  public async getComments(bugId: number) {
```

Comment:
The naming of this is likely to lead to some confusion, can we call it `getBugComments`?

Reply:
Renamed.